shared/metrics/src/main/java/io/pravega/shared/metrics/MetricsProvider.java

```diff
@@ -9,21 +9,15 @@
  */
 package io.pravega.shared.metrics;
 
-import com.codahale.metrics.Metric;
-import com.codahale.metrics.MetricRegistry;
-import com.google.common.annotations.VisibleForTesting;
+import io.micrometer.core.instrument.MeterRegistry;
+import io.micrometer.core.instrument.Metrics;
 
 public class MetricsProvider {
 
-    static final MetricRegistry METRIC_REGISTRY = new MetricRegistry();
+    static final MeterRegistry METRIC_REGISTRY = Metrics.globalRegistry;
```

Comment:
Why do you have a static final member that is just an instance of another static final member of another class?